pkg/render/render.go

```diff
@@ -236,12 +235,7 @@ func Agent(ctx context.Context, db kclient.Client, agent *v1.Agent, oauthServerU
 		return nil, nil, err
 	}
 
-	oauthEnv, err := OAuthAppEnv(ctx, db, agent.Spec.Manifest.OAuthApps, agent.Namespace, oauthServerURL)
-	if err != nil {
-		return nil, nil, err
-	}
-
-	extraEnv = append(extraEnv, oauthEnv...)
+	extraEnv = append(extraEnv, fmt.Sprintf("OAUTH_SERVER_URL=%s", oauthServerURL))
```

Comment:
dropping all the existing env url that we are passing down to the oauth tool. It is no longer needed(used to determine whether tool need to do oauth or not) and oauth tool can figure out the url by itself given the server url. https://github.com/obot-platform/tools/pull/536/commits/156b7bd1aac216b8ca612f9eeca7653fd6e65e8e#diff-9ae2ae74ccd09b47849da90bbe83ab989b5cf7bdc4d074c82046370972269defR78